package dmGuiDDF;

import "ddf/ddf_extensions.proto";

option java_package = "com.dynamo.gui.proto";
option java_outer_classname = "Gui";
option (ddf_java_package) = "com.dynamo.gui.ddf";

message SceneDesc
{
    message FontDesc
    {
        required string Name = 1;
        required string Font = 2;
        required uint32 MaxShapes = 3;
    }

    required string Script = 1;
    repeated FontDesc Fonts = 2;
}
